### DISCUSSION TITLE
'node' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by ucodetech** [spatie/laravel-pdf#121] [Q&A]
The command "node "C:\Users\username\Documents\opt\laravel\ProfjectName\vendor\spatie\browsershot\src/../bin/browser.cjs" "{""url"":""file:\/\/C:\\Users\\username\\AppData\\Local\\Temp\\2049785017-0761462001712764735\\index.html"",""action"":""pdf"",""options"":{""args"":[],""viewport"":{""width"":800,""height"":600},""displayHeaderFooter"":false,""printBackground"":true}}"" failed. Exit Code: 1(General error) Working directory: C:\Users\username\Documents\opt\laravel\ProjectName\public Output: ================ Error Output: ================ 'node' is not recognized as an internal or external command, operable program or batch file.

Please how do i fix this error, I only get it when i try to generate PDF.  already node is installed and added to part . am running node v20

**Comment by Professor93M** [comment]
You can remove the **vendor** folder and run the `composer install` command.

**Reply by ucodetech** [reply]
My complete vendor folder or spatie/browsershot

**Reply by ucodetech** [reply]
I have done that and it didnt work
